src/JestExt.ts

```diff
@@ -455,22 +503,6 @@ export class JestExt {
     this.jestProcessManager.stopAll()
   }
 
-  private getJestVersion(version: (v: number) => void) {

```

Comment:
Jest's --showConfig got only added in version 20.0.0, that's probably why we still relied on this `getJestVersion`. With your PR there would be no situation in which the "This extension relies on Jest 20+ features […]" warning gets displayed.
Since v20.0.0 is already older than one year, I wonder, if we could drop support for v19 (stopping to display the warning message)?